src/components/Icon/index.tsx

```diff
@@ -45,12 +44,12 @@ export default class Icon extends React.Component<IconProps, {}> {
 
   public render () {
     const { size, icon } = this.props;
-
-    return (
-      <svg className={this.getClasses()} style={this.getInlineStyles()} aria-hidden="true">
-        <use xlinkHref={`#${icon}`} />
-      </svg>
-    );
+    return React.createElement(ICONS[icon], {
```

Comment:
Do we need `React.createElement` here, or could we do something like this?

```
const CurrentIcon = ICONS[icon];
return <CurrentIcon className={} style={} />
```

Reply:
I thought I had tried that! But now it works.